src/guides/v2.3/security/google-recaptcha.md

```diff
@@ -12,6 +12,29 @@ Google reCAPTCHA provides a greater level of security for both the storefront an
 
 Google reCAPTCHA reduces potential user error when entering a Captcha code and encourages cart conversion without adding hurdles during checkout.
 
+## Types of Google reCAPTCHA
+
+There are three different types of reCAPTCHA available. They are,
+
+-  reCAPTCHA v3
+-  reCAPTCHA v2 ("I'm not a robot" Checkbox)
+-  reCAPTCHA v2 (Invisible reCAPTCHA badge)
+
+### reCAPTCHA v3
+
+reCAPTCHA v3 allows you to verify if an interaction is legitimate without any user interaction. It is a pure JavaScript API returning a score, giving you the ability to take action in the context of your site. For instance, requiring additional factors of authentication, sending a post to moderation, or throttling bots that may be scraping content.
```

Comment:
Rewrite the first sentence so that it doesn't contain two instances of the word "interaction". Maybe change the first instance to "a request".